src/configuration.ts

```diff
@@ -1,88 +1,130 @@
-import { Environment, FormatEnum, IPrecomputedBandit, PrecomputedFlag } from './interfaces';
+import {
+  Environment,
+  FormatEnum,
+  IObfuscatedPrecomputedBandit,
+  IPrecomputedBandit,
+  PrecomputedFlag,
+} from './interfaces';
 import {
   generateSalt,
-  getMD5Hash,
-  obfuscatePrecomputedBandits,
+  obfuscatePrecomputedBanditMap,
   obfuscatePrecomputedFlags,
 } from './obfuscation';
-import { ContextAttributes } from './types';
+import { ContextAttributes, MD5String } from './types';
+
+export interface IPrecomputedConfiguration {
+  // `format` is always `PRECOMPUTED`
+  readonly format: FormatEnum;
+  readonly obfuscated: boolean; // Always false.
```

Comment:
minor: in TS, we can do this trick:
```suggestion
  readonly format: FormatEnum.PRECOMPUTED;
  readonly obfuscated: true;
```

Reply:
thanks!